spec/examples/util/initTestI18n.js

```diff
@@ -0,0 +1,34 @@
+import {init} from 'i18next';
+import formatFunc from '../../../src/util/i18nFormatting';
+
+// There is an issue when loading json directly that gives an error
+// saying it is not valid javascript.  I believe it may be due to
+// reusing the webpack config object, but I'm not sure.
+// https://github.com/webpack-contrib/json-loader/issues/13
+
+// eslint-disable-next-line import/no-webpack-loader-syntax
+import enTestResourceFile from '!json-loader!./test-translation.json';
```

Comment:
So, why use a JSON file at all? Seems like for the purposes of testing, we can just create the test data in code?